tests/integ/test_dataframe.py

```diff
@@ -5180,3 +5181,30 @@ def test_create_dataframe_x_string_y_integer(session):
         Row(X="b", Y=2),
     ]
     assert result == expected_rows
+
+
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="File data type is not supported in Local Testing",
+)
+def test_create_dataframe_file_type(session, resources_path):
+    stage_name = Utils.random_name_for_temp_object(TempObjectType.STAGE)
+    _ = session.sql(f"create or replace temp stage {stage_name}").collect()
```

Comment:
```suggestion
    session.sql(f"create or replace temp stage {stage_name}").collect()
```
probably don't need the _ =

Reply:
done